recipes-security/refpolicy/refpolicy-mcs-2.%/patches/policy.modules.system.xen.diff

```diff
@@ -420,6 +420,7 @@
 +        atapi_helper_signal(xend_t)
 +        audio_helper_domtrans(xend_t)
 +        audio_helper_signal(xend_t)
++        varstored_domtrans(xend_t)
```

Comment:
Shouldn't this be varstored_watch_domtrans(xend_t) now?  Without this, you still were running varstored-watch as varstored_t, so you probably missed some new rules.